developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -1794,7 +1797,7 @@ KMX_DWORD GetXStringImpl(PKMX_WCHAR tstr, PFILE_KEYBOARD fk, PKMX_WCHAR str, KMX
     case 99:
       if (tokenFound) break;
       {
-        sprintf(ErrExtraLIB,"token: %c",(int)*p);
+        snprintf(ErrExtraLIB, 256, "token: %c",(int)*p);
```

Comment:
Shall we add a #define for these as well?

Reply:
+1

Reply:
done